application/Program.cs

```diff
@@ -12,7 +12,7 @@ public static void Main(string[] args)
 
 		    using (var amz = new AmazonSynchronizer(new Configuration()))
 		    {
-                amz.WithInterval(TimeSpan.FromMinutes(1))
+                amz.WithInterval(TimeSpan.FromMinutes(5))
```

Comment:
Hmm, is the `TimeSpan` passed in here not the amount of time between Amazon queries (to prevent getting rate limited by Amazon)?